rts/lra/lra-service-base/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>lra-parent</artifactId>
+    <groupId>org.jboss.narayana.rts</groupId>
+    <version>5.9.3.Final-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>lra-service-base</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.microprofile.lra</groupId>
+      <artifactId>microprofile-lra-api</artifactId>
+      <version>${version.microprofile.lra.api}</version>
+    </dependency>
+    
+    <!-- jboss logging -->
+    <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging-processor</artifactId>
+      <version>${version.org.jboss.logging.jboss-logging-processor}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging</artifactId>
+      <version>${version.org.jboss.logging.jboss-logging}</version>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <finalName>LRA Service base</finalName>
```

Comment:
Isn't this element used as the final name the build artefact, in which case we need a better name (in particular no spaces or capitals). Easiest is to just follow the name of the maven module that builds the artefact.

Also it isn't clear what this module is for so can you add a description element for the module.

Reply:
you are right, I will update this